R_scripts/licordat.R

```diff
@@ -57,27 +60,46 @@ apple <- read_dir("/Users/penn529/Desktop/apple_data/")
 x <- read_dir("/Users/penn529/Desktop/PREMIS/licor_test_data/")
 read_dir("/Users/penn529/Desktop/PREMIS/licor_test_data/SJ_6_1_16/")
 plant <- read_dir("/Users/penn529/Desktop/plant_data/")
+licorDat <- read_dir("/Users/penn529/Documents/GitHub/PREMIS-ghg/licor_data/")
 
 # Create practice df with multiple collars
 # merge with cores_collars.csv
 # practice plotting with ggplot
 d <- data.frame(Timestamp = rep(1:5, times = 12), Flux = runif(120), Collar = rep(1:120, each = 5))
-collardata <- read.csv("../design/cores_collars.csv")
-y <- merge(d, collardata)
+collarDat <- read.csv("/Users/penn529/Desktop/PREMIS/cores_collars.csv")
+y <- merge(d, collarDat)
```

Comment:
I'd suggest picking either `dplyr::xxx_join` or `merge` and sticking with it. So here, maybe change to `left_join` if that's what we're going to use. (This would be my preference.)

Reply:
I agree, line 64 down in licordat.R should have just been deleted since it had already been copied over to licorplot.R and fixed.